src/assets/i18n/en-us.json

```diff
@@ -75,6 +75,7 @@
   "delivered": "Delivered",
   "returned": "Returned",
   "inProgress": "In Progress",
+  "inSendProgress": "In Progress",
```

Comment:
From users' aspect, there's no way to tell the difference between `inSendProgress` and `inProgress` status from the same text, which makes this change fails to fix issue #199.